lib/manageiq/appliance_console/prompts.rb

```diff
@@ -168,6 +168,25 @@ def ask_for_disk(disk_name, verify = true)
       disk
     end
 
+    # Used for internal database configuration. Disk can be nil but in that case,
+    # our default pg mount point must already been manually mounted to a empty
+    # disk by user. Just return nil here and check in internal database config
+    def ask_for_disk_silently(disk_name)
```

Comment:
Couldn't we just add a `silent = false` parameter to `ask_for_disk` then change the caller that needs this to `ask_for_disk("database disk", false, true)`?

Then the silent parameter would govern whether we print the prompt on line 152. I think that does exactly what this method does.